### ISSUE TITLE
huggingface token

### ISSUE THREAD

**Issue by Anonym0us33** [NickLucche/stable-diffusion-nvidia-docker#28]
Is there any way to use this without a hugging face token, or to change the SD model to something besides 1.5? Or is it part of the docker script?
Could you possibly edit the docker script so that it no longer requires a HF download?

**Comment by NickLucche** [comment]
Yes this is definitely doable but currently it requires you to set the `-e MODEL_ID <model_name>` environment variable before starting the container to load a different model. Currently the version on master defaults to `stabilityai/stable-diffusion-2-base`.
I will look into making this more user-friendly.

>Could you possibly edit the docker script so that it no longer requires a HF download?
This can be done already by mounting hf cache directory, see here https://github.com/NickLucche/stable-diffusion-nvidia-docker/tree/master#about-model-versions.

**Comment by luffyx10** [comment]
> Yes this is definitely doable but currently it requires you to set the `-e MODEL_ID <model_name>` environment variable before starting the container to load a different model. Currently the version on master defaults to `stabilityai/stable-diffusion-2-base`. I will look into making this more user-friendly.
> 
> > Could you possibly edit the docker script so that it no longer requires a HF download?
> > This can be done already by mounting hf cache directory, see here https://github.com/NickLucche/stable-diffusion-nvidia-docker/tree/master#about-model-versions.

Hi, Thank you for this and I have a problem loading other models 
can you please make a detailed instruction because am really lost 
Thanks again
